content/installation/java/Java-linux-packages.md

```diff
@@ -0,0 +1,122 @@
+<!--
+title: "Java Agent Installation via Host Installer"
+description: "Overview of Java agent installation with Host Installer"
+tags: "installation Java agent linux Host Installer"
+-->
+
+## About Installation via System Package Managers
+
+The Contrast Java agent may be installed using system package managers. Unlike the *jar* file you can download from the Contrast UI, the Contrast Java agent *jar* installed by system package managers isn't preconfigured with connection parameters; instead, you must configure the Java agent using a [configuration file](installation-javaconfig.html).
+
+### The Exec Helper package 
+
+Some System Administrators may also want install the optional Exec Helper package for the Contrast Java agent, which configures the system to always enable Contrast whenever you execute `java`. Instead of a typical Contrast installation that requires users to configure their Java services to include the Contrast Java agent, the `contrast-java-agent-exec-helper` package intercepts all new `java` processes, and automatically configures the process to use Contrast. 
+
+This option is ideal for some scenarios in which Contrast must support legacy infrastructure, and your Java application server configuration isn't well understood. But, in most scenarios, administrators want more granular control over their Contrast configurations.
+
+## Installation for Ubuntu-Based Systems
+
+To install the Contrast Java agent with Ubuntu-based systems, complete the following steps. 
+
+* Configure your system to Contrast's distribution-agnostic Debian repository.
+
+```
+curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | sudo apt-key add -
+echo "deb https://contrastsecurity.jfrog.io/contrastsecurity/debian-staging/ all contrast" | sudo tee /etc/apt/sources.list.d/contrast-staging.list
+```
+
+* Once you've finished configuration, install the Contrast Java agent.
+
+```
+sudo apt-get update && sudo apt- get install `contrast-java-agent`
+```
+
+* Edit the */etc/contrast/contrast-java-config.yaml* file to configure Contrast Java agent to connect to the Contrast application. <!-- (Go to the YAML [Configuration doc](_________) for more info.) -->
+
+> **Note:** To complete the agent configuration process, follow the instructions for the [container](installation-javainstall.html) of your choice. 
```

Comment:
Is this clear enough? If not, we can reword and/or mention it in the intro paragraph.

Reply:
I want to better distinguish configuring the agent (i.e. `/etc/contrast/contrast-java-config.yaml`) from configuring the users' applications to include the agent. Instead of starting off with

> To complete the agent configuration process,

Can we change that clause to

> To include the Contrast Java agent in your Java applications,

Reply:
"container" can mean so many things in this space; I think we should qualify this term with "Java application container"